packages/instantsearch.js/src/connectors/menu/connectMenu.ts

```diff
@@ -298,12 +304,20 @@ const connectMenu: MenuConnector = function connectMenu(
               ? facetValues.data
               : [];
 
+          if (showMore) {
+            const showMoreTotalCount = Math.min(
+              showMoreLimit,
+              facetItems.length
+            );
+            showMoreCount = showMoreTotalCount - currentLimit;
```

Comment:
I wonder if the number given to the template shouldn't be `showMoreTotalCount`, as that's the number of items that will be visible. Not "show x more items", but "10 items / 13 items". You can still easily calculate "total - current" to get the difference. What do you think?